confluentinc/jmx-monitoring-stacks
# Add Stray Partitions Count to Grafana dashboards

This will add Stray Partitions Count to grafana dashboards.

How to test:

- run dev-toolkit: example ./start.sh --profile schema-registry --profile ksqldb --profile replicator --profile consumer
- connect to grafana: localhost:3000
- open "Kafka Cluster" dashboard
- See the next counter in Healthcheck section; warning is set to 40, fatal to 100
